dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -116,6 +116,8 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
             AppNotificationTelemetry::LogRegister(hr, m_appId);
         }) };
 
+        // Need to clear the RoActivateInstance caching for LRP proxyStub to be found.
```

Comment:
LRP references being made again here are unclear. Also hy should it be cleared? Is it a bug? If so, what is the bug number?
I would also move this outside of AppNotification APIs if possible. Is it possible to do it during install time?

Reply:
Fixed LRP reference. Location discussion in other thread.